packages/devtools/.eslintrc.js

```diff
@@ -1,185 +1,185 @@
 const rules = {
-  "arrow-body-style": "error",
-  "arrow-parens": ["off", "as-needed"],
-  camelcase: "error",
-  complexity: "off",
-  "constructor-super": "error",
-  curly: "error",
-  "dot-notation": "error",
-  "eol-last": "off",
-  eqeqeq: ["error", "smart"],
-  "guard-for-in": "error",
-  "id-blacklist": [
-    "error",
-    "any",
-    "Number",
-    "number",
-    "String",
-    "string",
-    "Boolean",
-    "boolean",
-    "Undefined"
+  'arrow-body-style': 'error',
+  'arrow-parens': ['off', 'as-needed'],
+  camelcase: 'error',
+  complexity: 'off',
```

Comment:
Is now a good time to fix the `'key':` vs `key:` discrepancy?

Reply:
JavaScript allows us to define keys without quotes surrounding them if they are single words.
So `key: "something"` is valid while `this-key: "something"` is not.
The valid way is `"this-key": "something"`

Reply:
...so do you think we should have all keys in double quotes? That way all keys have the same format.

Reply:
It is formatted automatically by Prettier. I don't think there is a config for it.
Take a look at this discussion:
https://github.com/prettier/prettier/issues/838

Reply:
Approved since this issue is not caused by this PR.